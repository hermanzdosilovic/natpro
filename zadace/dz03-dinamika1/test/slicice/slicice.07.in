60 10 2
410
834
984
174
567
134
649
196
825
312
181
482
805
707
316
884
607
460
879
401
1000
406
584
873
313
595
413
221
336
162
867
248
560
944
160
748
406
701
850
647
350
527
654
405
379
340
783
643
189
583
351
477
577
830
654
838
557
451
163
122
