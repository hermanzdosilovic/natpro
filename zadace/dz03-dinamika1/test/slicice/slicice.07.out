mirko
slavko
slavko
slavko
slavko
mirko
mirko
mirko
slavko
slavko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
slavko
mirko
mirko
mirko
mirko
slavko
mirko
mirko
mirko
mirko
slavko
slavko
slavko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
slavko
slavko
mirko
mirko
slavko
mirko
slavko
mirko
slavko
slavko
mirko
mirko
slavko
mirko
mirko
mirko
mirko
mirko
